libs/domain/picking/src/components/picking-list-card/picking-list-card.component.ts

```diff
@@ -1,16 +1,98 @@
-import { property } from 'lit/decorators.js';
+import { resolve } from '@spryker-oryx/di';
+import { PickingListService } from '@spryker-oryx/picking';
+import { html, LitElement, TemplateResult } from 'lit';
+import { property, state } from 'lit/decorators.js';
+import { when } from 'lit/directives/when.js';
 import type { PickingList } from '../../models';
+import { styles } from './picking-list-card.styles';
 
 export class PickingListCardComponent extends LitElement {
-  @property() pickingList?: PickingList;
+  static styles = styles;
 
-  protected override render(): unknown {
-    return html`<p>
-      PickingList ${this.pickingList?.id} in status ${this.pickingList?.status}
-      with ${this.pickingList?.items.length} item(s)
-    </p>`;
+  protected pickingListService = resolve(PickingListService);
+
+  @property({ type: Object }) pickingList?: PickingList;
+  @state() isDisabled?: boolean;
+
+  protected startPicking(): void {
+    this.isDisabled = true;
+    // TODO: push to customer note page if this.pickingList?.cartNote exists
+    // else
+    this.pickingListService.startPicking(this.pickingList!);
+    // TODO: push to picking-items page
+
+    this.isDisabled = false;
   }
-}
 
-export default PickingListCardComponent;
+  protected showCustomerNote(): void {
+    this.dispatchEvent(
+      new CustomEvent('showCustomerNote', {
+        detail: {
+          customerNote: this.pickingList?.cartNote,
+        },
+      })
+    );
+  }
+
+  protected override render(): TemplateResult {
+    return html`
+      <oryx-card>
+        <div slot="heading" class="qqq">qqq</div>
+        <div slot="heading" class="time">
+          ${this.formatTime(this.pickingList?.createdAt)}
+        </div>
+        <div slot="heading" class="identifier">${this.pickingList?.id}</div>
+
+        <div class="total">
+          <oryx-icon type="cart"></oryx-icon>
+          <span :data-e2e="order-total-label-${this.pickingList?.id}"
+            >${this.pickingList?.items?.length} items</span
+          >
+        </div>
+
+        ${when(
+          this.pickingList?.cartNote,
+          () => html`
+            <oryx-icon-button size="large">
+              <button
+                aria-label="Show customer note"
+                :data-e2e="order-note-button-${this.pickingList?.id}"
+                @click=${this.showCustomerNote}
+              >
+                <oryx-icon type="info"></oryx-icon>
+              </button>
+            </oryx-icon-button>
+          `
+        )}
+
+        <oryx-button
+          slot="footer"
+          type="primary"
+          size="large"
+          @click=${this.startPicking}
+        >
+          <button
+            :disabled=${this.isDisabled}
+            :data-e2e="start-picking-button-${this.pickingList?.id}"
+          >
+            Start picking
+          </button>
+        </oryx-button>
+      </oryx-card>
+    `;
+  }
+
+  protected formatTime(time?: Date): string {
+    if (!time) {
```

Comment:
We've already have and issue related to optional time: https://spryker.atlassian.net/browse/HRZ-2382

I believe here will be the same: render the formatted time or nothing. Don't forgot to update styles if need